developer_manual/core/theming.rst

```diff
@@ -172,9 +150,7 @@ If you want to override the translation of the term "Download" within the
     "nplurals=2; plural=(n != 1);"
   );
 
-Additionally you need to create another file
-``themes/THEME_NAME/apps/files/l10n/de.json`` with the same translations that
-look like this:
+Finally, you need to create another file ``owncloud/app/theme-example/l10n/de.json`` with the same translations that look like this:
```

Comment:
Same here, the directory would be `owncloud/apps/theme-example/apps/files/l10n/de.json` or actually `de_DE.json`.